### ISSUE TITLE
Next.js build fails when forced --bun (jest worker, retry limit)

### ISSUE THREAD

**Issue by Arctomachine** [oven-sh/bun#11961]
### What version of Bun is running?

1.1.14

### What platform is your computer?

Microsoft Windows NT 10.0.19045.0 x64

### What steps can reproduce the bug?

`bunx create-next-app` (accept defaults) and `bun run --bun build` (or `bun run --bun next build`)

### What is the expected behavior?

Finish build script and exit with code 0

### What do you see instead?

```
bun run --bun build
$ next build
  ▲ Next.js 14.2.4

   Creating an optimized production build ...

> Build error occurred
1 | (()=>{"use strict";var e={914:e=>{e.exports=(e,t=process.argv)=>{const r=e.startsWith("-")?"":e.length===1?"-":"--";const i=t.indexOf(r+e);const s=t.indexOf("--");return i!==-1&&(s===-1||i<s)}},460:(e,t,r)=>{Object.defineProperty(t,"__esModule",{value:true});t["default"]=void 0;var i=_interopRequireDefault(r(552));var s=r(582);function _interopRequireDefault(e){return e&&e.__esModule?e:{default:e}}function _defineProperty(e,t,r){if(t in e){Object.defineProperty(e,t,{value:r,enumerable:true,configurable:true,writable:true})}else{e[t]=r}return e}class Farm{constructor(e,t,r={}){var s,o;_defineProperty(this,"_computeWorkerKey",void 0);_defineProperty(this,"_workerSchedulingPolicy",void 0);_defineProperty(this,"_cacheKeys",Object.create(null));_defineProperty(this,"_locks",[]);_defineProperty(this,"_offset",0);_defineProperty(this,"_taskQueue",void 0);this._numOfWorkers=e;this._callback=t;this._computeWorkerKey=r.computeWorkerKey;this._workerSchedulingPolicy=(s=r.workerSchedulingPolicy)!==null&&s!==void 0?s:"roun | ... truncated 

error: Jest worker encountered 1 child process exceptions, exceeding retry limit
      at initialize (project\node_modules\next\dist\compiled\jest-worker\index.js:1:11584)
      at _onExit (project\node_modules\next\dist\compiled\jest-worker\index.js:1:12545)
      at emit (node:events:180:48)
      at #handleOnExit (node:child_process:594:16)

error: script "build" exited with code 1
```

### Additional information

Happens only on my computer, other people report that it works on theirs.
Happens on react 18 and 19, next 14 and 15, with and without react compiler enabled and plugin installed.
Basically it happens on fresh install without any changes.

**Comment by hspak** [comment]
This happens on GCP Cloudbuild running `bun run next build` with `oven/bun:1.1.18` and `next 14.2.4` as well.

**Comment by hspak** [comment]
FWIW, I just checked on v1.1.26 and my issue is gone.

**Comment by Arctomachine** [comment]
Works for me too, although 1 time of 3 tries it still gave this error
